### ISSUE TITLE
Join DigitalOcean Marketplace for one click deploy

### ISSUE THREAD

**Issue by alexanderadam** [kindmetrics/kindmetrics#50]
Another issue for better visibility. It seems that it needs those things to be fullfilled:

* [ ]  Automate building a single node image for droplet on DigitalOcean
* [ ]  Get approved to be added as partner on DigitalOcean Marketplace
* [ ]  Submit Image for approval
* [ ]  Image approved
* [ ]  Test one click deploy

**PS:** [I found this idea in the PostHog issues](https://github.com/PostHog/posthog/issues/905)

**Comment by confact** [comment]
I like this idea @alexanderadam 

My only concern is all environment variables. I have to handle them in a good way here.
I will  look into this more in detail to see if I can find a good solution to this and what alternatives I have.

Linode seems to have a nice solution for this so there it should be easy.
Heroku have some sort of parameter support but need to have some default value set.

But as I said, will have to look into this more. But this is surely something I want to add. It will make Kindmetrics more out there :)

**Comment by alexanderadam** [comment]
Yes, I understand that. It probably needs some sane defaults that can be overridden in the GUI or so.
This way it would probably even be a simple solution for people that have no knowledge about IT.

**Comment by alexanderadam** [comment]
Maybe also adding a bash script and mentioning a Docker one liner would be great.
Something like [Offen analytics](https://www.offen.dev/try-demo/).
